cartoframes/contrib/vector.py

```diff
@@ -26,24 +26,28 @@ def __init__(self, query, color=None, size=None, time=None):
         if self.time:
             self.styling += '\nfilter: torque($cartodb_id, 40, fade(0.1, 0.3))'
 
-def _get_html_doc(sources, creds=None, local_sources=None, basemap=None):
+def _get_html_doc(sources, bounds, creds=None, local_sources=None, basemap=None):
     html_template = os.path.join(
         os.path.dirname(__file__),
         '..',
         'assets',
         'vector.html'
     )
+
     with open(html_template, 'r') as html_file:
         srcdoc = html_file.read()
 
     if basemap is None:
         basemap = 'DarkMatter'
     credentials = {} if creds is None else dict(user=creds.username(), api_key=creds.key())
+
+
     return (
         srcdoc\
             .replace('@@SOURCES@@', json.dumps(sources))
             .replace('@@BASEMAPSTYLE@@', basemap)
             .replace('@@CREDENTIALS@@', json.dumps(credentials))
+            .replace('@@BOUNDS@@', bounds)
```

Comment:
This should be a `json.dumps(bounds)` to ensure it's converted as we expect

Reply:
Okay, but I don't understand why? `bounds` is a string type object already (just like basemap)